test/models/mainstream_browse_page_test.rb

```diff
@@ -26,6 +26,28 @@
     it "returns the browse page description" do
       assert_equal "Information about eligibility, claiming and when Child Benefit stops", @page.description
     end
+
+    describe "#second_level_pages_curated?" do
+
+      it "returns true when second_level_ordering == curated" do
+        @api_data["details"]["second_level_ordering"] = "curated"
+        assert_equal true, @page.second_level_pages_curated?
+      end
+
+      it "returns false when second_level_ordering == alphabetical" do
+        @api_data["details"]["second_level_ordering"] = "alphabetical"
+        assert_equal false, @page.second_level_pages_curated?
+      end
+
+      it "returns false when second_level_ordering is unspecified" do
+        @api_data["details"] = {}
+        assert_equal false, @page.second_level_pages_curated?
+      end
+
+      it "returns false when details hash is missing" do
+        assert_equal false, @page.second_level_pages_curated?
```

Comment:
You need an `@api_details.delete("details")` here because the details is set to empty hash in the test setup.